### ISSUE TITLE
Add tutorial links alongside tools/libraries/etc.

### ISSUE THREAD

**Issue by dsasmblr** [dsasmblr/game-hacking#6]
Where applicable, leave tutorial links to show people how to use a tool/library/etc.!

**Comment by FreeER** [comment]
Hm, is that really necessary for most of these, especially the AIOs and large libraries where no one tutorial is ever going to cover everything (and you're unlikely to get a comprehensive playlist either)...

If there was a simple/automatic way to add a link to google with the name + " tutorial" then I could understand that (doubt it's possible with md though) but otherwise... except for ones that don't give a tutorial on the download page or the first page of a google search, is it really worth the effort?

**Comment by dsasmblr** [comment]
The "where applicable" is key. =) No, it won't be feasible for some things, but if I find or make a tutorial that I feel fits well, I'll add it.

**Comment by FreeER** [comment]
Pretty sure a tutorial would be technically "applicable" to literally everything... even if it's just "here's how to download the file into the appropriate folder" (which, yes, some people seem to need a tutorial for) lol

The only real exceptions I can imagine are for website links like CEF/FRF (and maybe the books), and theoretically even those could have tutorials for how to register an account and what form of formatting the editors provide :D

So... kind of a question of how far down this rabbit hole do you want to slide :) Once you know that you could request that anyone wanting to add their own tools also find or create an appropriate tutorial to go with it and/or have a list for anyone bored to go through and supply.

**Comment by dsasmblr** [comment]
Yes, a tutorial can be literally anything. Because this has made you get this granular with it (not your fault; I understand you're not a mind-reader, lol), I'm just going to go ahead and nix this as an issue so as to prevent others from spinning their wheels as such. This was meant primarily for me as more of a note to myself that, if I saw what seemed to be a good cursory and/or generally comprehensive overview of something. I would link to it. Ultimately, this repo is going to have to be reformatted and prettified (which I've created a project for and will expand upon) no matter what. I'd like for that version to somehow be inclusive of copious tutorials, however it happens. The scope of that is beyond my current intentions for the repository.
